README.md

```diff
@@ -1,10 +1,10 @@
-# Feldera: Incremental Computation Made Easy
+# Feldera: Incremental Computation Tailored To Your Data
```

Comment:
Copy the "About" and say "The Feldera Incremental Computation Engine" till we find something better.